cbmc/proofs/IotMqtt_RemoveSubscriptionByTopicFilter/Makefile

```diff
@@ -0,0 +1,42 @@
+ENTRY=IotMqtt_RemoveSubscriptionByTopicFilter
+
+# We abstract this function because manual inspection demonstrates that it is unreachable
+ABSTRACTIONS += --remove-function-body _topicFilterMatch
+
+# We abstract all unreachable functions to improve coverage metrics
+ABSTRACTIONS += --remove-function-body _packetMatch
```

Comment:
Is this unreachable somehow different from the prior unreachable?

Reply:
Not really, I'll merge them together